site/docs/reference/Connectors/materialization-connectors/starburst.md

```diff
@@ -0,0 +1,136 @@
+# Starburst
+
+This connector materializes transactionally Flow collections into Iceberg or Delta Lake tables using AWS S3 storage in [Starburst Galaxy](https://www.starburst.io/platform/starburst-galaxy/).
+Starburst Galaxy connector supports only standard(merge) updates.
+
+The connector makes use of S3 AWS storage for storing temporarily data during the materialization process.
+
+[`ghcr.io/estuary/materialize-starburst:dev`](https://ghcr.io/estuary/materialize-starburst:dev) provides the latest connector image. You can also follow the link in your browser to see past image versions.
+
+## Prerequisites
+
+To use this connector, you'll need:
+
+* A Starburst Galaxy account (To create one: [Staburst Galaxy start](https://www.starburst.io/platform/starburst-galaxy/start/) that includes:
+  * A running cluster containing an [Amazon S3](https://docs.starburst.io/starburst-galaxy/working-with-data/create-catalogs/object-storage/s3.html) catalog
+  * A [schema](https://docs.starburst.io/starburst-galaxy/data-engineering/working-with-data-lakes/table-formats/index.html#create-schema) which is a logical grouping of tables
+  * Storage on S3 for temporary data with `awsAccessKeyId` and `awsSecretAccessKey` which should correspond to used catalog
+  * A user with a role assigned that grants access to create, modify, drop tables in specified Amazon S3 catalog
+* At least one Flow collection
+
+### Setup
+
+To get host go to your Cluster -> Connection info -> Other clients ([Connect clients](https://docs.starburst.io/starburst-galaxy/working-with-data/query-data/connect-clients.html))
+
+There is also need to grant access to temporary storage (Roles and privileges -> Select specific role -> Privileges -> Add privilege -> Location). "Create schema and table in location" should be selected. [Doc](https://docs.starburst.io/starburst-galaxy/cluster-administration/manage-cluster-access/manage-users-roles-and-tags/account-and-cluster-privileges-and-entities.html#location-privileges-)
+
+## Configuration
```

Comment:
Should call this setup for Estuary. Or call the other one configuration. Trying to be consistent in terminology for setups steps for both products.